test/unit/preprocessing/Discrimation_remover_optimized_preprocess/test_optimPreproc.py

```diff
@@ -0,0 +1,144 @@
+import pandas as pd
```

Comment:
nit: folder name has a typo (discrimination) and should start with lower case, file name should be snake case

Reply:
changed to lower case, thank you!